lib/linter/report-translator.js

```diff
@@ -115,6 +115,17 @@ function normalizeReportLoc(descriptor) {
     return descriptor.node.loc;
 }
 
+/**
+ * Check that a fix has a valid range.
+ * @param {Fix|null} fix The fix to validate.
+ * @returns {void}
+ */
+function validateFix(fix) {
+    if (fix) {
+        assert(typeof fix.range[0] === "number" && typeof fix.range[1] === "number", `Fix has invalid range: ${fix.range}`);
```

Comment:
The range itself is probably not enough detail for the error message. Can we use the full fix operation details?

Reply:
Sure. Is JSON.stringify the best way to log the full object here?